### PR TITLE
add BasicStructure

### PR DESCRIPTION
**Pull request by MarcRaeder**
add Basic Structure

### CODE REVIEW

File: my-app/src/Components/TypingArea.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Button } from "./Button";
+import { TextLengthInput } from "./TextLengtInput";
```
Comment (matthiasdanne) [inline my-app/src/Components/TypingArea.js]:
```suggestion
import { TextLengthInput } from "./TextLengthInput";
```

**Review by matthiasdanne — APPROVED** [review]

File: my-app/src/Components/Stats.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export function Stats(props) {
+  return <div class="StatsArea__Stats">{props.text}</div>;
+}
```
Comment (stefanluth) [inline my-app/src/Components/Stats.js]: import React :)

File: my-app/src/Components/TextLengthInput.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export function TextLengthInput() {
+  return (
+    <div class="TypingArea__TextLength">
+      Text Length
+      <input type="number" id="TextLength" onkeypress="handleKeyPress(event)" placeholder="Text-Length 1-100" />
+    </div>
+  );
+}
```
Comment (stefanluth) [inline my-app/src/Components/TextLengthInput.js]:
```suggestion
export function TextLengthInput(props) {
  return (
    <div class="TypingArea__TextLength">
      Text Length
      <input type="number" id="TextLength" onkeypress={props.handleKeyPress} placeholder="Text-Length 1-100" />
    </div>
  );
}
```

Use the props :)

Comment (stefanluth) [reply my-app/src/Components/TextLengthInput.js]: I can't find the handleKeyPress function :o

**Review by stefanluth — CHANGES_REQUESTED** [review]

File: my-app/src/Components/GenerateNewText.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  // current = document.createElement("SPAN");
+  // current.className = "TypingArea__Text--current";
+  // current.textContent = rest.textContent[0];
+  // rest.textContent = rest.textContent.slice(1);
+  // textarea = document.getElementsByClassName("TypingArea__Text");
+  // textarea[0].insertBefore(current, rest);
+  // document.querySelector(".TypingArea__input_error").innerHTML = "";
+  // incorrectKey = 0;
+  // document.getElementById("TextLength").blur();
+  // document.getElementById("btn").blur();
```
Comment (stefanluth) [inline my-app/src/Components/GenerateNewText.js]: Remove comments :)

File: my-app/src/Components/GenerateNewText.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+export async function GenerateNewText(textLength) {
+  const fehlermeldung = "Es sind nur 1 - 100 Wörter erlaubt";
```
Comment (stefanluth) [inline my-app/src/Components/GenerateNewText.js]:
```suggestion
  const errorMessage = "Es sind nur 1 - 100 Wörter erlaubt";
```

File: my-app/src/Components/GenerateNewText.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  if (textLength < 1 || textLength > 100) {
+    return <div className="Fehlermeldung">{fehlermeldung}</div>;
+  }
+  let file = "https://random-word-api.herokuapp.com/word?number=" + textLength;
+  Math.floor(Math.random() * 100);
+  let response = await fetch(file);
+  let words = await response.json();
+  let text = words.join(" ");
```
Comment (stefanluth) [inline my-app/src/Components/GenerateNewText.js]: Use `const` :)

File: my-app/src/Components/Stats.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import React from "react";
+export function Stats(props) {
```
Comment (stefanluth) [inline my-app/src/Components/Stats.js]:
```suggestion
import React from "react";

export function Stats(props) {
```

File: my-app/src/Components/TextLengthInput.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+export function TextLengthInput(props) {
```
Comment (stefanluth) [inline my-app/src/Components/TextLengthInput.js]:
```suggestion
import React from "react";

export function TextLengthInput(props) {
```

File: my-app/src/Components/TypingArea.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+import { Button } from "./Button";
+import { TextLengthInput } from "./TextLengthInput";
+import { TypingField } from "./TypingField";
+
+export function TypingArea() {
+  return (
+    <div className="TypingArea">
+      <h1 className="TypingArea__Header ">Typing 100</h1>
+      <TextLengthInput />
+      <TypingField />
```
Comment (stefanluth) [inline my-app/src/Components/TypingArea.js]: `TypingField` expects `props.text` :)

File: my-app/src/index.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <>
+      <TypingArea />
+      <StatsArea />
+    </>
+  );
+}
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(TypingPractice());
```
Comment (stefanluth) [inline my-app/src/index.js]:
```suggestion
root.render(<TypingPractice />);
```

**Review by stefanluth — CHANGES_REQUESTED** [review]

**Review by stefanluth — APPROVED** [review]

**Pull request merged**